apps/sms/test/unit/thread_ui_test.js

```diff
@@ -6945,6 +6952,24 @@ suite('thread_ui.js >', function() {
       // Status checking for bubble select mode enabled
       assert.isFalse(threadMessagesClass.contains('editable-select-mode'));
       assert.isTrue(range > 0);
+      sinon.assert.calledWithMatch(
+        ThreadUI.container.removeEventListener,
+        'contextmenu',
+        ThreadUI
+      );
```

Comment:
wondering if it's really useful with the next test? I think the next test is more meaningful (with the event being sent)